mesonbuild/interpreter.py

```diff
@@ -388,6 +389,12 @@ def get(self, name):
     def keys(self):
         return self.held_object.values.keys()
 
+    def append_method(self, args, kwargs):
```

Comment:
need @FeatureNew and @noKwargs annotations.

Reply:
I amended the commit adding these annotations (hopefully well, I just looked at how these are used elsewhere).